applications/GeoMechanicsApplication/custom_conditions/thermal_point_flux_condition.h

```diff
@@ -0,0 +1,68 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//
+//  Main authors:    Mohamed Nabi
+//                   John van Esch
+//
+
+#pragma once
+
+#include "custom_conditions/T_condition.h"
+#include "includes/serializer.h"
+
+namespace Kratos {
+
+template <unsigned int TDim, unsigned int TNumNodes>
+class KRATOS_API(GEO_MECHANICS_APPLICATION) GeoThermalPointFluxCondition
+    : public GeoTCondition<TDim, TNumNodes> {
+public:
+    using GeometryType   = Geometry<Node>;
+    using PropertiesType = Properties;
+    using NodesArrayType = GeometryType::PointsArrayType;
+
+    KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(GeoThermalPointFluxCondition);
+
+    GeoThermalPointFluxCondition();
+
+    GeoThermalPointFluxCondition(IndexType NewId, GeometryType::Pointer pGeometry);
+
+    GeoThermalPointFluxCondition(IndexType NewId,
+                            GeometryType::Pointer pGeometry,
+                            PropertiesType::Pointer pProperties);
+
+    ~GeoThermalPointFluxCondition() override;
+
+    Condition::Pointer Create(IndexType NewId,
+                              NodesArrayType const& rThisNodes,
```

Comment:
Nitpicking: we always put `const` in front of the type name rather than after it. So:
```suggestion
                              const NodesArrayType& rThisNodes,
```